run_all_tools.sh

```diff
@@ -24,38 +81,9 @@ export AWS_REGION=us-west-1
-find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do echo $test_case ; ORG_PATH=$PWD ; cd $test_case ; if [ ! -f checkov_results.txt ]; then docker run -t -v $PWD:/tf bridgecrew/checkov --quiet -d /tf | sed 's/\[[0-9;]*m//g' | sed "s~$ORG_PATH~tool-compare~" > checkov_results.txt ; fi ; cd $ORG_PATH; done
-
-# tfsec
-echo Now running tfsec on all cases
-docker pull tfsec/tfsec:latest
-docker run -t -v $PWD:/tf tfsec/tfsec --version > version_tfsec.txt
-find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do echo $test_case ; ORG_PATH=$PWD ; cd $test_case ; if [ ! -f tfsec_results.txt ]; then docker run --rm -v "$(pwd):/src" tfsec/tfsec /src --no-color | sed "s~$ORG_PATH~tool-compare~" > tfsec_results.txt ; fi; cd $ORG_PATH; done
-
-# KICS
-echo Now running KICS on all cases
-docker pull checkmarx/kics:latest
-docker run -t -v $PWD:/tf checkmarx/kics version | awk '{print $NF}' > version_kics.txt
-find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do echo $test_case ; ORG_PATH=$PWD ; cd $test_case ; if [ ! -f kics_results.txt ]; then docker run --rm -v "$(pwd):/src" checkmarx/kics:latest scan -p /src | sed "s~$ORG_PATH~tool-compare~" | grep -v "Executing queries" > kics_results.txt ; fi ; cd $ORG_PATH; done
-
-# Terrascan
-echo Now running Terrascan on all cases
-brew install terrascan
-terrascan version | awk '{print $NF}' > version_terrascan.txt
-find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do echo $test_case ; ORG_PATH=$PWD ; cd $test_case ; if [ ! -f terrascan_results.txt ]; then terrascan scan | sed "s~$ORG_PATH~tool-compare~" > terrascan_results.txt ; fi ; cd $ORG_PATH; done
-
-# Snyk
-echo Now running Snyk on all cases
-docker pull snyk/snyk-cli:docker
-docker run -t -v empty:/project -e SNYK_TOKEN snyk/snyk-cli:docker --version | tail -n 1 > version_snyk.txt
-find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do echo $test_case ; ORG_PATH=$PWD ; cd $test_case ; if [ ! -f snyk_results.txt ]; then docker run --rm -v "$(pwd):/project" -e SNYK_TOKEN snyk/snyk-cli:docker iac test --experimental /project | sed "s~$ORG_PATH~tool-compare~" > snyk_results.txt ; fi ; cd $ORG_PATH; done
-
-# Cloudrail
-echo Now running Cloudrail on all cases
-docker pull indeni/cloudrail-cli:latest
-docker run -t -v $PWD:/tf indeni/cloudrail-cli --version | awk '{print $NF}' > version_cloudrail.txt
-find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do echo $test_case ; ORG_PATH=$PWD ; cd $test_case ; if [ ! -f cloudrail_results.txt ]; then docker run --rm -v $PWD:/data -e CLOUDRAIL_API_KEY indeni/cloudrail-cli run --tf-plan plan.out --output-file cloudrail_results.txt --no-cloud-account --auto-approve -v; fi ; cd $ORG_PATH; done
+run_checkov
```

Comment:
The code structure is a bit odd. how about adding the calling of all functions as a default in the `case` statement you have? (use `*)`)